collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+#### Navigation
+
+Exciting additions have been cooking in the navigation system of the engine by Godot's resident navigation maintainer [smix8](https://github.com/smix8). At the top of the list is navigation mesh baking for 2D, bringing it to parity with 3D navigation ([GH-80796](https://github.com/godotengine/godot/pull/80796)). It is capable of handling physics bodies, mesh instances, plain polygons, and, of course, tilemaps ([GH-82465](https://github.com/godotengine/godot/pull/82465)).
+
+On top of that, both 2D and 3D navigation servers now support multi-threading when baking the navigation mesh ([GH-79972](https://github.com/godotengine/godot/pull/79972)), removing stutters and bringing performance to a whole new level.
+
+#### Tilemaps
+
+One of the areas that receive most feedback is the tiles/tilemap system and editor. And so improvements to them in Godot 4.2 are plentiful, targeting both usability and performance.
+
+Let's start with [Gilles Roudière](https://github.com/groud) who has laid a lot of ground work for future improvements with an internal refactoring ([GH-78328](https://github.com/godotengine/godot/pull/78328)) earlier this year. This has been followed by two major performance optimizations, cutting down update times multiple times over — thanks to smart tile grouping using quadrants ([GH-81070](https://github.com/godotengine/godot/pull/81070)) and Y-sorting upgrade ([GH-73813](https://github.com/godotengine/godot/pull/73813)), as well as other optimizations.
+
+Impressive as that is, it might just be overshadowed by the shear number of usability improvements to the creation and utilization of tilemaps contributed by [Tomasz Chabora](https://github.com/KoBeWi). Probably the most notable addition in this release is the new tool which allows you to flip and rotate any tile or tile pattern when placing them in the world ([GH-80144](https://github.com/godotengine/godot/pull/80144)). And then the list continues with improvements to scene tiles, polygon editing, various hints and tips that direct users, and so on and so forth ([GH-77986](https://github.com/godotengine/godot/pull/77986), [GH-79285](https://github.com/godotengine/godot/pull/79285), [GH-79512](https://github.com/godotengine/godot/pull/79512), [GH-79676](https://github.com/godotengine/godot/pull/79676), [GH-79899](https://github.com/godotengine/godot/pull/79899), [GH-79904](https://github.com/godotengine/godot/pull/79904), [GH-80754](https://github.com/godotengine/godot/pull/80754), ...).
```

Comment:
```suggestion
Impressive as that is, it might just be overshadowed by the shear number of usability improvements to the creation and utilization of tilemaps contributed by [Tomasz Chabora](https://github.com/KoBeWi). Probably the most notable addition in this release is the new tool which allows you to flip and rotate any tile or tile pattern when placing them in the world ([GH-80144](https://github.com/godotengine/godot/pull/80144)). The list continues with improvements to scene tiles, polygon editing, various hints and tips that direct users, and so on and so forth ([GH-77986](https://github.com/godotengine/godot/pull/77986), [GH-79285](https://github.com/godotengine/godot/pull/79285), [GH-79512](https://github.com/godotengine/godot/pull/79512), [GH-79676](https://github.com/godotengine/godot/pull/79676), [GH-79899](https://github.com/godotengine/godot/pull/79899), [GH-79904](https://github.com/godotengine/godot/pull/79904), [GH-80754](https://github.com/godotengine/godot/pull/80754), ...).
```

Reply:
Any particular reason for that?

Reply:
Having *both* "And" and "then" seemed redundant. I considered suggesting changing to either "And the list continues with" or "Then the list continues with", but actually thought it was sounded better without either of them! If you don't like it without both, then "And the list continues" with would be my second-choice recommendation.

Reply:
```suggestion
Impressive as that is, it might just be overshadowed by the shear number of usability improvements to the creation and utilization of tilemaps contributed by [Tomasz Chabora](https://github.com/KoBeWi). Probably the most notable addition in this release is the new tool which allows you to flip and rotate any tile or tile pattern when placing them in the world ([GH-80144](https://github.com/godotengine/godot/pull/80144)). And the list continues with improvements to scene tiles, polygon editing, various hints and tips that direct users, and so on and so forth ([GH-77986](https://github.com/godotengine/godot/pull/77986), [GH-79285](https://github.com/godotengine/godot/pull/79285), [GH-79512](https://github.com/godotengine/godot/pull/79512), [GH-79676](https://github.com/godotengine/godot/pull/79676), [GH-79899](https://github.com/godotengine/godot/pull/79899), [GH-79904](https://github.com/godotengine/godot/pull/79904), [GH-80754](https://github.com/godotengine/godot/pull/80754), ...).
```